src/integration-test/java/mil/dds/anet/emails/AnetEmailWorkerIT.java

```diff
@@ -0,0 +1,110 @@
+package mil.dds.anet.emails;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.beans.AnetEmail;
+import mil.dds.anet.config.AnetConfiguration;
+import mil.dds.anet.config.AnetConfiguration.SmtpConfiguration;
+import mil.dds.anet.database.EmailDao;
+import mil.dds.anet.threads.AnetEmailWorker;
+import mil.dds.anet.utils.EmailResponse;
+import mil.dds.anet.utils.FakeSmtpServer;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({EmailDao.class, AnetConfiguration.class, ScheduledExecutorService.class,
+    AnetObjectEngine.class})
+public class AnetEmailWorkerIT {
+
+  AnetEmailWorker emailWorker;
+  EmailDao emailDao;
+  FakeSmtpServer emailServer;
+  AnetObjectEngine instance;
+
+  /**
+   * Sets up the test.
+   */
+  @Before
+  public void setUp() {
+    emailServer = new FakeSmtpServer();
+
+    emailDao = PowerMockito.mock(EmailDao.class, Mockito.RETURNS_MOCKS);
+    AnetConfiguration config = PowerMockito.mock(AnetConfiguration.class, Mockito.RETURNS_MOCKS);
+    ScheduledExecutorService scheduler =
```

Comment:
Make these `final`.

Reply:
Since we (rightly) enforce the use of the final keyword, I propose to also apply it to method parameters. It will not change anything for the caller, but it may let us spot any potential problems. We could even automaticattly apply it through the 'spotlessJavaApply' Gradle command.